scripts/Strategies/SimpleMACD.py

```diff
@@ -59,7 +59,7 @@ def find_trade_signal(self, broker, epic_id):
         px['macd'] = (px['12_ema'] - px['26_ema'])
         px['macd_signal'] = px['macd'].rolling(9).mean()
         px['positions'] = 0
-        px['positions'][9:]=np.where(px['macd'][9:]>=px['macd_signal'][9:],1,0)
+        px.loc('positions', 9:) = np.where(px.loc('macd', 9:) >= px.loc('macd_signal', 9:) , 1, 0)
```

Comment:
px.loc[9:, ('positions')] = np.where(px.loc[9:, ('macd')] >= px.loc[9:, ('macd_signal')] , 1, 0)

Reply:
I think square brackets are correct but the order should be [row, col], so like before?
```suggestion
        px.loc['positions', 9:] = np.where(px.loc['macd', 9:] >= px.loc['macd_signal', 9:] , 1, 0)
```

Reply:
I think that `positions` is a column in the dataframe as `macd` and `macd_signal`.
I tried `px.loc['positions', 9:]....` and it complains about 'macd' not being the index of the dataframe: `'the label [macd] is not in the [index]'`
I guess the dataframe is composed like this:
```
  macd  macd_signal  positions
0  x       x          x
1  x       x          x
2  x       x          x
```

Reply:
This then
```suggestion
        px.loc[9:, 'positions'] = np.where(px.loc[9:, 'macd'] >= px.loc[9:, 'macd_signal'] , 1, 0)
```